ci/gpu/build.sh

```diff
@@ -61,6 +61,11 @@ gpuci_conda_retry install -c conda-forge -c rapidsai -c rapidsai-nightly -c nvid
       "rapids-doc-env=${MINOR_VERSION}.*" \
       "shap>=0.37,<=0.39"
 
+gpuci_conda_retry remove -c conda-forge -c rapidsai -c rapidsai-nightly -c nvidia \
```

Comment:
Everything here looks great! I think this is the only "testing" code that needs to be remove now that the integration PR is in, right? Otherwise, I think we're ready to merge.

Reply:
Yes, I'm about to remove it, since the `integration` PR is now merged.